managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -157,7 +157,7 @@ public class ManagedLedgerImpl implements ManagedLedger, CreateCallback {
     protected Map<String, String> propertiesMap;
     protected final MetaStore store;
 
-    final ConcurrentLongHashMap<CompletableFuture<ReadHandle>> ledgerCache =
+    final ConcurrentLongHashMap<CompletableFuture<ReadHandle>> ledgerHandleCache =
```

Comment:
Why do you change this?

Reply:
In order to express the meaning more clearly. 
you are right. it should separate another PR for this change.
updated.